bin/pre-commit-hook.js

```diff
@@ -8,7 +8,8 @@ const gitFiles = execSync( 'git diff --cached --name-only --diff-filter=ACM' )
 	.toString()
 	.split( '\n' )
 	.map( name => name.trim() );
-const jsFiles = gitFiles.filter( name => name.endsWith( '.js' ) || name.endsWith( '.jsx' ) );
+const jsFiles = gitFiles.filter( name => name.startsWith( '_inc' ) )
```

Comment:
That may not be enough I am afraid, as we have old js files in the `_inc` directory as well. `_inc/twitter-timeline.js` is a good example.

Maybe we should limit this to `_inc/client`?

Reply:
That makes sense. Updated to `_inc/client` in bbe8f53